html/signupsignin.html

```diff
@@ -52,6 +46,26 @@ <h2>Sign in</h2>
             </form>
         </div>
     </div>
+
+    <script src="signupsignin.js"></script>
```

Comment:
the same here, this file would not be found under this location because you moved it to js forlder.

Reply:
Connected them now.